cob_object_detection_msgs/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ add_service_files(
     AcquireObjectImage.srv
     BagTrainObject.srv
     BaTestEnvironment.srv
+    ComputeGraspsVacuumGripper.srv
```

Comment:
Do you really want a manipulation-related message in cob_object_detection_msgs?
Isn't this rather application-specific?